ext/bg/js/translator.js

```diff
@@ -70,8 +70,8 @@ class Translator {
         return {sequencedDefinitions, defaultDefinitions};
     }
 
-    async getMergedSecondarySearchResults(text, expressionsMap, secondarySearchTitles) {
-        if (secondarySearchTitles.length === 0) {
+    async getMergedSecondarySearchResults(text, expressionsMap, secondarySearchDictionaries) {
+        if (secondarySearchDictionaries.length === 0) {
```

Comment:
`secondarySearchDictionaries.size`

Reply:
I was almost positive I had missed one of these, but I didn't see it when doing a check. Thanks for noticing.